### PR TITLE
Simplifying image names and hippocampus surface generation.

### PR DESCRIPTION
**Pull request by rphellan**
Simplifying image names to make them more understable.
Adding hippocampus surface generation option.

### CODE REVIEW

File: ipl/lp/pipeline.py [hunk 1]
```diff
@@ -994,7 +999,21 @@ def standard_pipeline(info,
...
                             iter_summary['skin_surface'] = t1w_tal_noscale_skin
 
-                                
+                        if surfaces_parameters.get('hippocampus',False) :
+                            with mincTools(verbose=2) as minc:
+                                #start with t1w_tal_noscale and then segment hippocampus
+                                tmp_work = minc.tmp('tmp_work')
+                                tmp_output = minc.tmp('tmp_output')
+                                fusion_segment(input_scan= t1w_tal_noscale.scan, 
+                                            library_description='/data/ipl/scratch08/vfonov/adni_jens/jens_hc_lib_20170621/library.json',
```
Comment (vfonov) [inline ipl/lp/pipeline.py:1011]: `/data/ipl/scratch08/vfonov/adni_jens/jens_hc_lib_20170621` - refers to internal BIC path, you should specify location of the library as parameter
also, library_description should be a dictionary, to load from json:
`json.load(open('.......json'))`

File: ipl/lp/pipeline.py [hunk 2]
```diff
@@ -994,7 +999,21 @@ def standard_pipeline(info,
...
+                                            output_segment=tmp_output,
+                                            parameters='/data/ipl/scratch08/vfonov/adni_jens/jens_hc_segment_20170621.json',
```
Comment (vfonov) [inline ipl/lp/pipeline.py:1013]: parameters should be dictionary too

File: ipl/segment/fuse.py [hunk 3]
```diff
@@ -33,8 +33,8 @@
 
 
 def fusion_segment( input_scan,
-                    library_description,
                     output_segment,
+                    library_description={},
```
Comment (vfonov) [inline ipl/segment/fuse.py]: This change will break other modules that call fusion_segment

File: ipl/lp/pipeline.py [hunk 4]
```diff
@@ -582,12 +585,13 @@ def standard_pipeline(info,
             t1w_tal=MriScan(prefix=tal_dir, name='tal_'+dataset_id, modality='t1w')
             t1w_tal_fld=MriScan(prefix=tal_dir, name='tal_fld_'+dataset_id, modality='t1w') # to xform nonuniformity correction field into stx space
 
-            t1w_tal_noscale=MriScan(prefix=tal_dir, name='tal_noscale_'+dataset_id,modality='t1w')
+            t1w_tal_noscale=MriScan(prefix=tal_dir, name=dataset_id+'_full_head_image',modality='t1w')
```
Comment (vfonov) [inline ipl/lp/pipeline.py:590]: This change breaks compatibility with other pipeline outputs

**Review by vfonov — CHANGES_REQUESTED** [review]
See my comments to the code

**Comment by rphellan** [comment]
I will just close this pull request as I am working on the IBIS branch. The changes can be ignored.

**Pull request closed without merging**